eZ/Bundle/EzPublishCoreBundle/Command/RegenerateUrlAliasesCommand.php

```diff
@@ -78,6 +78,11 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
                 'Only Locations with provided ID\'s will have URL aliases regenerated',
                 []
+            )->addOption(
+                'force',
+                'f',
+                InputOption::VALUE_NONE,
+                'Prevents confirmation dialog. Please use it carefully.'
```

Comment:
Let's be more precise:
```suggestion
                'Prevents confirmation dialog when used with --no-interaction. Please use it carefully.'
```
requesting because currently, when `--no-interaction` is not set, `--force` will have no effect - confirmation will be shown.

Reply:
Done in https://github.com/ezsystems/ezplatform-kernel/pull/19/commits/8d4b804fae8879673f5a7dd78300d0b2254e6b16